lib/media/segment_utils.js

```diff
@@ -91,6 +93,15 @@ shaka.util.SegmentUtils = class {
       return null;
     }
     const onlyAudio = hasAudio && !hasVideo;
+    const closedCaptions = new Map();
+    if (hasVideo) {
+      const captionParser = new shaka.media.ClosedCaptionParser('video/mp2t');
+      captionParser.parseFrom(data);
+      for (const stream of captionParser.getStreams()) {
+        closedCaptions.set(stream, stream);
```

Comment:
The JSDoc for the `closedCaptions` value in the manifest externs says:
```
A map containing the description of closed captions, with the caption
channel number (CC1 | CC2 | CC3 | CC4) as the key and the language code
as the value. If the channel number is not provided by the description,
we'll set an 0-based index as the key.
```
From what I can see, the string values that the CEA decoder returns from `getStreams()` are all caption channel numbers. But the language code is unknown when the information is found this way, right?
That externs JSDoc says what we do when the channel number is unknown, but does not mention what happens in this case. You should update it to include that info, if we really can't tell what language these captions are.

Reply:
Done!